### DISCUSSION TITLE
How to make Links and Backlinks showing up in search?

### DISCUSSION THREAD

**Discussion by szarazbalint** [nvim-telekasten/telekasten.nvim#260] [Q&A] [answered]
Hi!
The last picture in the readme shows the number of links and backlinks next to each search results. (L1 B3 - Note title...)
How do I make that happen? Is there some setting for it I'm missing?

**Comment by lambtho12** [comment, marked as answer]
Yes, the function that shows links and backlinks seems to have issues (#256). I do not think it is a problem on your side.

**Reply by szarazbalint** [reply]
Thanks for the reply!
